File: _프로그래머스 js/20. 최댓값과 최솟값.js
/*
* Title: 최댓값과 최솟값
https://school.programmers.co.kr/learn/courses/30/lessons/12939
*/
function solution(s) {
  const numls = s.split(' ').map((str) => Number(str));
  const min = Math.min(...numls);
  const max = Math.max(...numls);

  return `${min} ${max}`;
}

console.log(solution('1 2 3 4'));

File: 연습장/2308/10_목/20. 최댓값과 최솟값.py
"""
* Title: 최댓값과 최솟값
https://school.programmers.co.kr/learn/courses/30/lessons/12939
"""


def solution(s):
    strls = s.split()
    intls = []

    for i in range(0, len(strls)):
        intls.append(int(str[i]))
    intls.sort()
    min = str(intls[0])
    max = str(intls[len(intls) - 1])
    return min + ' ' + max
